src/runtime_src/core/common/utils.cpp

```diff
@@ -259,4 +260,32 @@ parse_clock_id(const std::string& id)
   return clock_str != clock_map.end() ? clock_str->second : "N/A";
 }
 
+uint64_t
+mac_addr_to_value(std::string mac_addr)
+{
+  boost::erase_all(mac_addr, ":");
+  return std::stoul(mac_addr, nullptr, 16);
```

Comment:
You forgot an l:
```suggestion
  return std::stoull(mac_addr, nullptr, 16);
```
This is needed since a long value on windows is 4 bytes and on linux it is 8 bytes, hence a long long is needed.